pom.xml

```diff
@@ -64,6 +65,16 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
what does this plugin do?

Reply:
it's maven's main (and advanced) test runner plugin. configuration was needed to make the tests run in parallel and in full isolation (due to some isolation issues w/ the static test integration layer)